docs/pages/video.js

```diff
@@ -366,7 +365,17 @@ function Example () {
 export async function getStaticProps(): Promise<{| props: {| generatedDocGen: DocGen |} |}> {
   const generatedDocGen = await docgen({ componentName: 'Video' });
 
+  generatedDocGen.props.ref = {
+    required: false,
+    defaultValue: null,
+    flowType: {
+      name: 'Ref',
+      raw: 'Ref<typeof Video>',
```

Comment:
Should "Video" be capitalized here? Are we actually referring to the `<video>` element?

Reply:
It's the Video class component. I think the correct type is more like `{ current: Node}`. I might fix it in the future. I wanted to document it. as it's not a forwardRef.